documentation/docs/specs/didcomm/protocols/authentication.md

```diff
@@ -35,7 +35,7 @@ This protocol allows two parties to mutually authenticate by disclosing and veri
 
 <div style={{textAlign: 'center'}}>
 
-<sub>For guidance on diagrams see the <a href="../overview.md#diagrams">corresponding section in the overview</a>.</sub>
```

Comment:
should this be with `.md`?

Reply:
Html is not transformed by mdx. So it‘s not needed here.